envs/stag-mwc.yaml

```diff
@@ -20,5 +20,6 @@ dependencies:
     - samtools =1.9
     - groot =0.8.4
     - pip
+    - matplotlib =2.2.2
```

Comment:
`matplotlib =3.1.1` is already present in the environment. Is there anything in v3.1.1 that doesn't work like you want it to?